doc/api/util.md

```diff
@@ -144,7 +144,7 @@ exports.obsoleteFunction = util.deprecate(() => {
 ```
 
 When called, `util.deprecate()` will return a function that will emit a
-`DeprecationWarning` using the `process.on('warning')` event. The warning will
+`DeprecationWarning` using the `'warning'` event. The warning will
```

Comment:
Nit: should we link `'warning'`? Otherwise it's not clear what is the target object.

Reply:
Done in the fixup commit.